docs/trees/guide.md

```diff
@@ -273,6 +265,16 @@ val right = q"Right()"
 println(q"$left + $right")
 ```
 
+> It's important to keep in mind that quasiquotes expand at compile-time into
+> as if you had written normal constructors by hand. This means for example
```

Comment:
could you please add commas on either side of `for example`? i know they weren't there before :)

Reply:
fixed! https://github.com/scalameta/scalameta/pull/3572/commits/28f654f220623965e0ebb431c0bd8edc0b3fd3b7